app/cdap/components/Replicator/Create/Content/SelectTables/index.tsx

```diff
@@ -706,4 +706,9 @@ class SelectTablesView extends React.PureComponent<ISelectTablesProps, ISelectTa
 
 const StyledSelectTables = withStyles(styles)(SelectTablesView);
 const SelectTables = createContextConnect(StyledSelectTables);
-export default SelectTables;
+
+// Higher Order Component wrapping class component so we can use useFeatureFlag hook
+export default ({ children, ...props }) => {
+  props.useReplicationTransformation = useFeatureFlag('replication.transformations.enabled');
+  return <SelectTables {...props} />;
```

Comment:
Nit: Should we make this
```
return <SelectTables
  {{
    ...props,
    useReplicationTransformation: useFeatureFlag('replication.transformations.enabled')
  }}
/>
```
?

I know it's short HOC but mutating `props` is still bugging me.

Reply:
Yeah absolutely I like that a lot better.